app/src/main/java/com/eventyay/organizer/core/event/create/EventDetailsStepThree.java

```diff
@@ -144,7 +147,10 @@ public void showProgress(boolean show) {
 
     @Override
     public void onSuccess(String message) {
-        Toast.makeText(getActivity(), message, Toast.LENGTH_SHORT).show();
+        if(!message.equals("")){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/app/fossasia/open-event-orga-app/pullRequest?prid=4568976)